client/extensions/woocommerce/state/sites/settings/stripe-connect-account/selectors.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import { get, omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const getRawSettings = ( state, siteId ) => {
+	return get( state, [
+		'extensions',
+		'woocommerce',
+		'sites',
+		siteId,
+		'settings',
+		'stripeConnectAccount',
+	] );
+};
+
```

Comment:
Always nice to have docblocks on selectors.

Reply:
Good idea - fixed in https://github.com/Automattic/wp-calypso/pull/19115/commits/a94094989d046c97bfe9a684c0d0f8b6a4ca411a